README.md

```diff
@@ -47,23 +47,23 @@ If there are any issues with the space or you are using a client that does not s
 
 We also have a subreddit you can post your issues and suggestions on:
 
-[r/PolyMCLauncher](https://www.reddit.com/r/PolyMCLauncher/)
+[r/PlaceholderMCLauncher](https://www.reddit.com/r/PlaceholderMCLauncher/)
```

Comment:
Non-existent. https://www.reddit.com/r/PlaceholderMC/ is taken by me, if anybody wants it